Remove clear, id and mknod

The base system will include utilchest and cbase that already includes
these binaries.

diff --git a/Makefile b/Makefile
index b526421..36e1332 100644
--- a/Makefile
+++ b/Makefile
@@ -39,10 +39,7 @@ LIB = $(LIBUTIL)
 BIN = \
 	blkdiscard        \
 	chvt              \
-	clear             \
 	ctrlaltdel        \
-	dd                \
-	df                \
 	dmesg             \
 	eject             \
 	fallocate         \
@@ -52,7 +49,6 @@ BIN = \
 	getty             \
 	halt              \
 	hwclock           \
-	id                \
 	insmod            \
 	killall5          \
 	last              \
@@ -60,8 +56,6 @@ BIN = \
 	login             \
 	lsmod             \
 	lsusb             \
-	mesg              \
-	mknod             \
 	mkswap            \
 	mount             \
 	mountpoint        \
@@ -70,7 +64,6 @@ BIN = \
 	passwd            \
 	pidof             \
 	pivot_root        \
-	ps                \
 	pwdx              \
 	readahead         \
 	respawn           \
@@ -87,27 +80,19 @@ BIN = \
 	unshare           \
 	uptime            \
 	vtallow           \
-	watch             \
-	who
+	watch

 MAN1 = \
 	chvt.1              \
-	clear.1             \
-	dd.1                \
-	df.1                \
 	dmesg.1             \
 	eject.1             \
 	fallocate.1         \
 	free.1              \
-	id.1                \
 	login.1             \
-	mesg.1              \
-	mknod.1             \
 	mountpoint.1        \
 	pagesize.1          \
 	passwd.1            \
 	pidof.1             \
-	ps.1                \
 	pwdx.1              \
 	respawn.1           \
 	stat.1              \
@@ -116,8 +101,7 @@ MAN1 = \
 	unshare.1           \
 	uptime.1            \
 	vtallow.1           \
-	watch.1             \
-	who.1
+	watch.1

 MAN8 = \
 	ctrlaltdel.8        \
